### PR TITLE
feat(RHTAPREL-172): add new rh-push-to-external-registry pipeline

### PR DESCRIPTION
**Pull request by scoheb**
- new pipeline to push snapshots to quay with no pyxis interactions

### CODE REVIEW

File: pipelines/rh-push-to-external-registry/README.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+| releaseplanadmission | The namespaced name (namespace/name) of the releasePlanAdmission | No | - |
+| snapshot | The namespaced name (namespace/name) of the snapshot | No | - |
+| enterpriseContractPolicy | JSON representation of the policy to be applied when validating the enterprise contract | No | - |
+| enterpriseContractPublicKey | Public key to use for validation by the enterprise contract | Yes | k8s://openshift-pipelines/public-key |
+| postCleanUp | Cleans up workspace after finishing executing the pipeline | Yes | true |
+| verify_ec_task_bundle | Switch back to using bundle resolvers for the verify-enterprise-contract task | No | - |
+| taskGitUrl | The url to the git repo where the release-service-catalog tasks to be used are stored | Yes | https://github.com/redhat-appstudio/release-service-catalog.git |
+| taskGitRevision | The revision in the taskGitUrl repo to be used | Yes | main |
+
+## Changes in 1.2.0
```
Comment (johnbieren) [inline pipelines/rh-push-to-external-registry/README.md:20]: Shouldn't be a changelog as it is a new pipeline

File: pipelines/rh-push-to-external-registry/README.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# Push to External Registry Pipeline
```
Comment (johnbieren) [inline pipelines/rh-push-to-external-registry/README.md:1]: This should be updated

File: pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+metadata:
+  name: rh-push-to-external-registry
+  labels:
+    app.kubernetes.io/version: "1.0.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton pipeline to release Snapshot to Quay
```
Comment (johnbieren) [inline pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml:13]: maybe? needs update

File: pipelines/rh-push-to-external-registry/README.md (hunk 1 above)
Comment (scoheb) [reply pipelines/rh-push-to-external-registry/README.md]: removed

File: pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml (hunk 3 above)
Comment (scoheb) [reply pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml]: updated. but let me know what I am missing or not saying?

File: pipelines/rh-push-to-external-registry/README.md (hunk 2 above)
Comment (scoheb) [reply pipelines/rh-push-to-external-registry/README.md]: updated. but let me know what I am missing or not saying?

Comment (johnbieren) [reply pipelines/rh-push-to-external-registry/README.md]: It was identical to the push-to-external-registry one. This is a different pipeline. It should have its own name

File: pipelines/rh-push-to-external-registry/README.md
Comment (mmalina) [inline pipelines/rh-push-to-external-registry/README.md:1]: I'm confused. I was wondering why `push-to-external-registry` does always put data into Pyxis, but this Red Hat specific one wouldn't. So I checked the Jira: https://issues.redhat.com/browse/RHTAPREL-172

And it says: "As an external, non-RH user..." So why is this prefixed `rh-`? Also looking at the comments in the Jira, shouldn't it be rather that we rename the current `push-to-external-registry` to `rh-push-to-external-registry` and then this new one without Pyxis will be `push-to-external-registry`? Also, can we not reuse the `release` pipeline as Johnny suggested there?

Comment (scoheb) [reply pipelines/rh-push-to-external-registry/README.md:1]: Yes. I got mixed up.

I will fix this PR

**Comment by scoheb** [comment]
Closing in favour of #302

**Pull request closed without merging**